cluster/examples/workloads/kubernetes/wordpress/aws/network-config/dbsubnetgroup.yaml

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
Wanted to get your thoughts on where these yaml config files should live. I am struggling with where to put them for the services guide and thought it might be nice if we just made the directory structure more in-line with the guides, i.e. `cluster/examples/guides/stacks/aws/...`. Alternatively, we could just put them actually in `docs/`. Any thoughts @negz @prasek @jbw976 ?

Reply:
@hasheddan I personally think having these in `examples/wordpress/aws` makes the most sense for easy discovery. I have no strong opinion about `/guides/stacks/aws/` path, but would rather not put them in `docs/` folder